models/config/binance_parser.py

```diff
@@ -77,13 +77,13 @@ def parser(app, binance_config, args={}):
 
     if 'api_key_file' in binance_config:
         try :
-            with open( binance_config['api_key_file'], 'r') as f :
+            with open(binance_config['api_key_file'], 'r') as f :
                 key = f.readline().strip()
                 secret = f.readline().strip()
             binance_config['api_key'] = key
             binance_config['api_secret'] = secret
         except :
-            raise RuntimeError('Unable to read ' + binance_config['api_key_file'])
+            raise RuntimeError('Unable to read' + binance_config['api_key_file'])
```

Comment:
your changes convert "Unable to read binance.key" into "Unable to readbinance.key" by removing the space. it is very unlikely this fixed anything.